manifests/t/TheDoggyBradSoftwareLabs/EdgescapeNavigator/2.0/TheDoggyBradSoftwareLabs.EdgescapeNavigator.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: TheDoggyBradSoftwareLabs.EdgescapeNavigator
+PackageVersion: "2.0"
+PackageLocale: en-US
+Publisher: TheDoggyBrad Software Labs
+PublisherUrl: https://github.com/thedoggybrad
+PackageName: Edgescape Navigator
+PackageUrl: https://github.com/thedoggybrad/edgescape_navigator
+License: MIT
+ShortDescription: Edgescape Navigator Setup

```

Comment:
Please provide a better description of what the application is, does, or is used for